src/hostname/hostnamed.c

```diff
@@ -601,6 +613,24 @@ static int property_get_hardware_model(
         return property_get_hardware_property(reply, userdata, PROP_HARDWARE_MODEL, get_hardware_model);
 }
 
+static int property_get_firmware_version(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+        _cleanup_free_ char *firmware_version = NULL;
```

Comment:
Please insert a blank line before the variable declaration if function arguments are wrapped.